src-java/base-topology/base-storm-topology/src/main/java/org/openkilda/wfm/share/mappers/DetectConnectedDevicesMapper.java

```diff
@@ -19,9 +19,10 @@
 import org.openkilda.model.DetectConnectedDevices;
 
 import org.mapstruct.Mapper;
+import org.mapstruct.ReportingPolicy;
 import org.mapstruct.factory.Mappers;
 
-@Mapper
+@Mapper(unmappedTargetPolicy = ReportingPolicy.ERROR)
```

Comment:
I think we can set the config globally. Add the following argument to the compiler -Amapstruct.unmappedTargetPolicy=ERROR